forge-gui/res/cardsfolder/g/gang_up.txt

```diff
@@ -0,0 +1,7 @@
+Name:Gang Up
+ManaCost:X B
+Types:Instant
+K:Assist
```

Comment:
@tehdiplomat will your current Assist wiring properly handle this {X} in the reminder text?

Reply:
We can test it. But i'm pretty sure Announce happens first so it should be fine.

Reply:
X works fine on my testing